tests/agent/test_agent_langchain_on_intel_hpu.sh

```diff
@@ -60,6 +81,28 @@ function start_tgi_service() {
     echo "Service started successfully"
 }
 
+function start_vllm_service() {
+    # redis endpoint
+    echo "token is ${HF_TOKEN}"
+
+    #single card
+    echo "start vllm gaudi service"
+    docker run -d --runtime=habana --rm --name "test-comps-vllm-gaudi-service" -p $vllm_port:80 -v $vllm_volume:/data -e HF_TOKEN=$HF_TOKEN -e HF_HOME=/data -e OMPI_MCA_btl_vader_single_copy_mechanism=none -e PT_HPU_ENABLE_LAZY_COLLECTIVES=true -e http_proxy=$http_proxy -e https_proxy=$https_proxy -e VLLM_SKIP_WARMUP=true --cap-add=sys_nice --ipc=host opea/vllm:hpu --model ${model} --host 0.0.0.0 --port 80 --block-size 128 --max-num-seqs  4096 --max-seq_len-to-capture 8192 --enable-auto-tool-choice --tool-call-parser mistral
```

Comment:
Does tool-call-parser=mistral work for llama models? Is there any limitation? I recall the parser can only parse 1 tool call. if there are 2+ tool calls, it will return a plain json string.

Reply:
For Llama, it should be llama3_json; I realized llama just is supported recently.
https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html

Reply:
@minmin-intel , I noticed that vllm-gaudi hasn't updated with this new llama3_json "tool-call-parser" yet. May add this support once vllm-fork code updated.